algorithmic_efficiency/workloads/mnist/workload.py

```diff
@@ -19,10 +26,18 @@ def loss_type(self):
 
   @property
   def num_train_examples(self):
-    return 60000
+    return 50000
 
   @property
-  def num_eval_examples(self):
+  def num_eval_train_examples(self):
+    return 10000
```

Comment:
I may be mistaken here, but I think currently, we are using `num_eval_train_examples = num_train_examples = 50000`, not `10000`, no?
Judging from:

https://github.com/znado/algorithmic-efficiency/blob/17fc48fd0062289f05c9eb62d1d45f72e5837d08/algorithmic_efficiency/workloads/mnist/mnist_pytorch/workload.py#L62-L63

we are using the same dataset for both `train` and `train_eval`.

If I didn't make a mistake, the same would be true for the `ImageNet` workload as well.